exhibitor-appliance.yaml

```diff
@@ -1,14 +1,21 @@
 SenzaInfo:
   StackName: exhibitor
   Parameters:
-    - DockerImage:
-        Description: Docker image of exhibitor
+    - DockerVersion:
+        Description: "Docker image version of exhibitor."
+    - DockerBaseImage:
```

Comment:
Following the docker nomenclature, *base image* makes me think in this:
https://docs.docker.com/engine/reference/glossary/#base-image

I understand why not to use `DockerImage` because of the change from the previous version, but what about using `DockerRepository` (see https://docs.docker.com/engine/reference/glossary/#repository)